ext/kv/01_db.ts

```diff
@@ -477,7 +485,7 @@ class AtomicOperation {
       core.serialize(message, { forStorage: true }),
       opts?.delay ?? 0,
       opts?.keysIfUndelivered ?? [],
-      null,
+      opts?.backoffSchedule ?? null,
```

Comment:
Also validate `backoffSchedule` similarly to `delay`?

Reply:
moved validation to js